Formula/fceux.rb

```diff
@@ -1,8 +1,7 @@
 class Fceux < Formula
   desc "All-in-one NES/Famicom Emulator"
   homepage "https://fceux.com/"
-  url "https://downloads.sourceforge.net/project/fceultra/Binaries/2.3.0/fceux-2.3.0.tar.gz"
-  sha256 "f166f2fea084162f5b823bbbc6843141e23908ae4ee69c8c4e0849b320d7a1dc"
+  url "https://github.com/TASVideos/fceux.git", using: :git, tag: "fceux-2.3.0", revision: "65c5b0d2a1c08db75bb41340bfa5534578926944"
```

Comment:
Why did you replace the tarball URL?

Reply:
During the build, there are some scripted git commands that are generate a cpp header file. that file was blank when the git rev info was not available. So using the official git tag instead. The GUI has an about fceux window that shows git url and rev details for traceability.

Reply:
```suggestion
  url "https://github.com/TASVideos/fceux.git",
      tag: "fceux-2.3.0",
      revision: "65c5b0d2a1c08db75bb41340bfa5534578926944"
```